src/primitives/block.cpp

```diff
@@ -25,9 +25,8 @@ std::string CBlock::ToString() const
         hashMerkleRoot.ToString(),
         nTime, nBits, nNonce,
         vtx.size());
-    for (unsigned int i = 0; i < vtx.size(); i++)
-    {
-        s << "  " << vtx[i]->ToString() << "\n";
+    for (const auto &tx : vtx) {
```

Comment:
> Hm, when I look at your last patch #10916 you actually do the same.

You mean using `const auto& var`, not `const auto &var`? Makes sense that I use what I advocate :)